test/basic/basic.go

```diff
@@ -93,6 +93,22 @@ func (s *TestSuite) TestBasics(t *testing.T) {
 	resp, err = subClient.Recv() // Now get the addition notification
 	assert.NoError(t, err)
 	assert.Equal(t, "201", resp.GetUpdate().Update[0].Path.Elem[1].Key["port"])
+
+	wg2 := sync.WaitGroup{}
+	wg2.Add(5) // setting counter to 5 (until at least 5 hosts are found..)
```

Comment:
Let's set the number to all the hosts that we're expecting to find, which is 20 (10 per each of the 2 leaves). The test will run a bit longer, but that's OK.

Reply:
addressed